fastlane/lanes/build.rb

```diff
@@ -359,4 +367,47 @@ def generate_prototype_build_number
       "#{branch}-#{commit}"
     end
   end
+
+  # Uploads the React Native JavaScript bundle and source map files.
+  # These files are provided by the Gutenberg Mobile library.
+  #
+  # @param [String] app App name, e.g. 'WordPress' or 'Jetpack'.
+  # @param [String] flavor Build flavor, e.g. 'Jalapeno' or 'Vanilla'.
+  # @param [String] build_type Build type, e.g. 'Debug' or 'Release'.
+  # @param [String] release_version Release version name to attach the files to in Sentry.
+  #
+  def upload_gutenberg_sourcemaps(app:, flavor:, build_type:, release_version:)
+    # Load Sentry properties
+    sentry_path = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'src', app, 'sentry.properties')
+    sentry_properties = JavaProperties.load(sentry_path)
+    sentry_token = sentry_properties[:"auth.token"]
+    project_slug = sentry_properties[:"defaults.project"]
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-Android/blob/f132e700785ba69cbb9320f8793e693509b3a5ba/fastlane/lanes/build.rb#L384"></span>Style/QuotedSymbols: Prefer single-quoted symbols when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>